index.html

```diff
@@ -0,0 +1,168 @@
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.3/css/all.min.css"
+      integrity="sha512-VtJLSzuz+PBooLZf5v5g5w5d5Hxu+7ySeNvYhXn1+cPyVBaLjZP+RQJff+FZYw8fj3PYqy+iyGG6dYU6W8lqTQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Lato:wght@400;700;900&display=swap"
+      rel="stylesheet"
+    />
+    <script
+      src="https://kit.fontawesome.com/51f27b15da.js"
+      crossorigin="anonymous"
+    ></script>
+    <title>Abrehot Library</title>
+  </head>
+  <body>
+    <header>
+      <nav class="star">
+        <i class="fa-sharp fa-solid fa-bars fa-xl hamburger"></i>
+      </nav>
+      <nav class="first-nav">
+        <ul>
+          <li>
+            <a href="#"><i class="fa-brands fa-facebook-f"></i></a>
+          </li>
+          <li>
+            <a href="#"><i class="fa-brands fa-twitter"></i></a>
+          </li>
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Logout</a></li>
+        </ul>
+      </nav>
+      <nav class="second-nav">
+        <div>
+          <a href="index.html"><h2>Abrehot Library</h2></a>
+        </div>
+        <div>
+          <ul>
+            <li><a href="about.html">About</a></li>
+            <li><a href="#">Books</a></li>
+            <li><a href="#">Membership</a></li>
+            <li><a href="#">Events</a></li>
+            <li><a href="#">Launched</a></li>
+            <li class="last-item"><a href="#">Library</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <!-- Mobile Menu -->
+    <section class="mobile-menu">
+      <button class="x-mark">&times;</button>
+      <ul id="menu-list">
+        <li><a id="" href="index.html">Home</a></li>
+        <li><a id="" href="about.html">About</a></li>
+        <li><a id="" href="#">Contact</a></li>
+      </ul>
+    </section>
+    <!-- Home page Intorduction -->
+    <main class="bg-gray">
+      <section class="introduction home-intro bg-gray font">
+        <h1>"Unleash your curiosity at the library."</h1>
+        <h2>ABREHOT Library</h2>
+        <div class="p-2 border-1 bg-gray">
+          <p class="p-1">
+            "Embark on a journey of discovery and endless learning as you
+            immerse yourself in the boundless treasures of knowledge and
+            imagination available at your fingertips in our library."
+          </p>
+        </div>
```

Comment:
- [OPTIONAL] Please make sure that your headline section elements have the same font `size, color, height, width, and spacing padding/spacing` as defined in the original design. cause you to need to stick the design as much as possible. 
 - [Optional] Also, for the background, I know you are trying to use a related image with the project, But keep in mind we need to deliver something identical. Hence It would be great to use a similar background image. 
### Your design 
![image](https://user-images.githubusercontent.com/69077061/235102895-d9073e0a-cb67-4993-bb5c-145ac55e9e30.png)

### Original design 
![198701404-f4138d1d-135f-4c01-aade-e1994a238fd1](https://user-images.githubusercontent.com/69077061/235102964-599e32db-7eb3-4fef-b153-56690674ee54.jpg)